docs/tasks/administer-cluster/weave-network-policy.md

```diff
@@ -103,8 +99,80 @@ spec:
 **Caution:** After applying the network policy, pods outside the namespace you specify may be unable to connect with pods inside the namespace.
 {. :caution}
 
-{% endcapture %}
+## Test the installation
+
+1. Verify that the weave works.
+
+Enter the following command:
+
+```shell
+kubectl get po -n kube-system -o wide
+```
+
+The output is similar to this:
+
+```
+NAME                                    READY     STATUS    RESTARTS   AGE       IP              NODE
+weave-net-1t1qg                         2/2       Running   0          9d        192.168.2.10    workndoe3
+weave-net-231d7                         2/2       Running   1          7d        10.2.0.17       worknodegpu
+weave-net-7nmwt                         2/2       Running   3          9d        192.168.2.131   masternode
+weave-net-pmw8w                         2/2       Running   0          9d        192.168.2.216   worknode2
+```
+
+Each Node has a weave Pod, and all Pods are `Running` and `2/2 READY`. (`2/2` means that each Pod has `weave` and `weave-npc`.)
+
+2. Create a Network Policy. 
+
+For more information, see "[this link]()".
```

Comment:
@dengyi1996 Please provide a link to the relevant documentation for creating a Network Policy.